cluster/src/main/scala/kanaloa/reactive/dispatcher/ClusterAwareBackend.scala

```diff
@@ -44,13 +45,18 @@ class ClusterAwareBackend(
   override def apply(f: ActorRefFactory): Future[ActorRef] = {
     val retriever = f.actorOf(ClusterAwareBackend.retrieverProps(router, routingLogic))
     import system.dispatcher
-    retriever.ask(ClusterAwareBackend.GetRoutee)(timeout = timeout.duration * 2).mapTo[RouteeRef].map(_.actorRef) //let the retriever handles the time internally.
-
+    //let the retriever handles the time internally.
```

Comment:
`handles` -> `handle`

also I feel like some of these lines weren't picked up by the autoformatter? might want to try reformatting.